system/iptables_raw.py

```diff
@@ -4,6 +4,7 @@
 """
 (c) 2016, Strahinja Kustudic <strahinjak@nordeus.com>
 (c) 2016, Damir Markovic <damir@damirda.com>
+(c) 2016, Allen Sanabria <asanabria@linuxdynasty.org>
```

Comment:
I never saw that anywhere. I contributed to a few modules and never changed the author tag, nor did I see that modules have a lot of authors listed. Only people who are listed are usually a few people who created the module or those who are constantly maintaining it.